src/main/java/svc/data/citations/datasources/tyler/TylerViolationStatus.java

```diff
@@ -0,0 +1,37 @@
+package svc.data.citations.datasources.tyler;
+
+import svc.models.VIOLATION_STATUS;
+
+public enum TylerViolationStatus {
```

Comment:
This enum I think has more logic than it really needs. It appears you're just string matching and doing the same work that Java will do for you. Check out what I did for the CITATION_DATASOURCE (I think that's what it's called) and the CitationDataSourceFactory